### PR TITLE
Allow JSON 2.0 on Ruby 2.2 and above

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,6 +8,11 @@ if Gem::Version.new(RUBY_VERSION) <= Gem::Version.new('1.9.3')
   gem 'net-ssh', '~> 2.9'
 end
 
+if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.2')
+  gem 'json', '< 2.0'
+  gem 'rack', '< 2.0
```
Comment (chris-rock) [inline Gemfile]: @jkeiser we have a small type here

**Comment by chris-rock** [comment]
Great addition @jkeiser Lets get this merged, once the syntax error is resolved

**Comment by jkeiser** [comment]
@chris-rock fixed! Awaiting appveyor/travis.

**Comment by chris-rock** [comment]
We are going to address the docker timeout issues (not related to this improvement) in another PR.

**Pull request merged**